src/docs/asciidoc/storage.adoc

```diff
@@ -556,8 +556,8 @@ the Hot Restart directory. For this, you need to perform the following steps:
 . Configure the Persistent Memory as a File System
 . Configure the Hot Restart Store to Use Persistent Memory
 
-Using this Persistent Memory, you can get the Hot Restart times approximately
-by 250%. You can find the configuration steps in the Hot Restart Store section
+Using Persistent Memory, Hot Restart times can be drastically improved.
```

Comment:
Apart from the grammar fix, I am not aware about the source of "250% improvement" statement but the claim seems broad and depends on hot restart store parallelism level. The ["Advancements in High Speed, In-Memory Systems" white paper](https://hazelcast.com/resources/advancements-in-high-speed-in-memory-systems/) includes a benchmark that demonstrates improvement up to ~3x faster in hot restart time. @Serdaro feel free to rephrase.